CONTRIBUTING.md

```diff
@@ -93,16 +102,25 @@ bool func(int param1, T param2) {
 
     return true;
 }
+} // namespace name
 
-/** Test function */
+/**
+ * @brief Test implementations
+ * @returns void
+ */
 static void test() {
     /* desciptions of the following test */
     assert(func(...) == ...); // this ensures that the algorithm works as expected
 
     // can have multiple checks
 }
 
-/** Main function */
+/**
+ * @brief Main function
+ * @param argc commandline argument count (ignored)
+ * @param argv commandline array of arguments (ignored)
+ * @returns 0 on exit
+ */
```

Comment:
This sounds a bit too verbose to me. As a reader, I'm not learning anything new and therefore it beats the purpose of having the comment in the first place.